crypto/src/fiat_shamir/transcript.rs

```diff
@@ -0,0 +1,70 @@
+use sha3::{Digest, Sha3_256};
+
+struct Transcript {
+    data: Vec<u8>,
```

Comment:
Is this expected to do anything other than hash the contents? If not, why not just keep a hasher and stream into it? It would probably be much more memory and CPU effective.

Reply:
Thanks! It looks like a good idea.